### PR TITLE
zh-cn: init the translation of API ref sidebars

### PR DESCRIPTION
**Pull request by yin1999**
### Description

init the translation of API ref sidebars

Upstream changes: mdn/content#29844

### CODE REVIEW

File: files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+此部分列出了你可以在 `GroupData` 条目中包含的所有子成员。
+
+注意，列在列出的子成员中的大部分值都等同于链接文本，以及附加到主 API 索引页面（`https://developer.mozilla.org/<language-code>/docs/Web/API`）的末尾的别名（slug）以创建显示的链接的最终 URL。因此，例如，“Response”将创建如下链接：
+
+```html
+<li><a href="/zh-CN/docs/Web/API/Response">Response</a></li>
+```
+
+这也有些例外。例如，“guides”子成员包含一个或多个链接信息（标题和别名），用于定义与指南/教程相关的链接。在这种情况下，别名将附加到 MDN 文档根路径（`https://developer.mozilla.org/_<language-code>/docs`）的末尾，并允许包含 MDN 上任何位置的文章。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md:63]: 应该是 `https://developer.mozilla.org/<language-code>` 末尾吧，比如下面这个

image

应该组成 `https://developer.mozilla.org/<language-code>/docs/Web/API/Geolocation_API/Using_the_Geolocation_API`

```suggestion
这也有些例外。例如，“guides”子成员包含一个或多个链接信息（标题和别名），用于定义与指南/教程相关的链接。在这种情况下，别名将附加到 MDN 文档根路径（`https://developer.mozilla.org/<language-code>/`）的末尾，并允许包含 MDN 上任何位置的文章。
```

**Review by jasonren0403 — COMMENTED** [review]
有一处建议

File: files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md:63]: Still need to do some investigation and update the English documentation. Let me fix the upstream doc again

Comment (yin1999) [reply files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md:63]: Found an exception in mdn/content, and the URL address may be broken, raise mdn/content#30485 to fix it.

Correct the description in: mdn/content#30484.

Comment (yin1999) [reply files/zh-cn/mdn/writing_guidelines/howto/write_an_api_reference/sidebars/index.md]: fixed

**Review by jasonren0403 — APPROVED** [review]
Thank you @yin1999 👍

**Pull request merged**